docs/index.rst

```diff
@@ -1,20 +1,23 @@
-Welcome to Kaolin's documentation!
-==================================
+Welcome to Kaolin Library Documentation
+=======================================
 
 .. image:: ../assets/kaolin.png
 
-NVIDIA Kaolin library provides a PyTorch API for working with a variety of 3D representations and includes a growing collection of GPU-optimized operations such as modular differentiable rendering, fast conversions between representations, data loading, 3D checkpoints and more.
+NVIDIA Kaolin library provides a PyTorch API for working with a variety of 3D representations and includes a growing collection of GPU-optimized operations such as modular differentiable rendering, fast conversions between representations, volumentric rendering, data loading, 3D checkpoints and more.
```

Comment:
s/volumentric rendering/volumetric rendering

Also we already have "differentiable rendering" doesn't that already include volumetric rendering?
And I think this should be consistent with the `README.md`

Reply:
Removed. We may want to update the list of features with more details.